<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="poorvid" failures="1" tests="3" name="DataDrivenDevelopement-1" time="76.337" errors="0" timestamp="2023-06-30T12:29:33 IST">
  <testcase classname="ExcelExample" name="demoClass" time="27.888"/>
  <testcase classname="ExcelExample" name="demoClass" time="22.507"/>
  <testcase classname="ExcelExample" name="demoClass" time="22.336">
    <failure type="java.lang.AssertionError" message="Invalid credentials , Login failed! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Invalid credentials , Login failed! expected [true] but found [false]
at ExcelExample.demoClass(ExcelExample.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- demoClass -->
</testsuite> <!-- DataDrivenDevelopement-1 -->
